package.json

```diff
@@ -29,8 +29,8 @@
     "test:changed": "lerna run --since origin/master --include-dependents test",
     "test:update": "lerna run test -- -u",
     "test:update:changed": "lerna run --since origin/master --include-dependents test -- -u",
-    "test:slow": "lerna run test:slow",
-    "test:slow:coverage": "lerna run test:slow --stream -- --collect-coverage --testPathIgnorePatterns",
+    "test:slow": "lerna run test:slow && bash ./ironfish-cli/scripts/import-export-test.sh",
+    "test:slow:coverage": "lerna run test:slow --stream -- --collect-coverage --testPathIgnorePatterns && bash ./ironfish-cli/scripts/import-export-test.sh",
```

Comment:
I think we should separate out these tests into a different test command. test:importexport?